packages/flutter_tools/templates/plugin/ios-objc.tmpl/Tests/PluginUnitTests.m.tmpl

```diff
@@ -0,0 +1,28 @@
+#import <XCTest/XCTest.h>
+#import "{{pluginClass}}.h"
+#import "{{pluginClass}}+Internal.h"
```

Comment:
Could you please elaborate why we need both "{{pluginClass}}.h" and "{{pluginClass}}+Internal.h" imported. It seems like "{{pluginClass}}+Internal.h" is created to only expose methods for testing?